packages/config/src/load.js

```diff
@@ -27,6 +30,21 @@ export async function loadNuxtConfig ({
     configFile = undefined
   }
 
+  // Load env
+  envConfig = defu(envConfig, {
+    dotenv: '.env',
+    systemVars: true,
```

Comment:
This needs to be made optional as it can be problematic in some setups with `pm2`.

I can't find the ref right now but apparently `process.env` has a limit! Couple it with a Kubernetes-based setup with hundreds of pre-populated environment vars and you're left scratching your head for a few days ;)

Reply:
I can't understand what's the _limit_? Nothing is exposed it is just an internal object used for expanding. If a k8s deployment is already having hundreds of envs, they are already in process.env

Reply:
Actually, sorry, this has no relation to the Nuxt usage. 

This is a problem specific to pm2 and is why I'm using a forked version of it. When forking a new process, it will inject the env for the process, and it will fail if it's "too big". How big it can be apparently varies in different platforms.

Still I think having `systemVars` be made optional might be something of interest to some users, would it be too hard to add it? Maybe expose dotEnvOptions?

Reply:
It can be optionally disabled but honestly there is no reason disabling by default. Adviced by both 12 factor and dotenv, environment variables have highest periority and for a container native deployment it is something obviously required ;)

Regarding pm2 issue, if process.env is already too big, this PR makes no difference. Flag only adds it to options._env nothing more 😀

Reply:
I will add cli flag too

Reply:
5cb822135d2e1b9011da9410f0cee8ba84a5178a